api/newsletters/newsletter_views.py

```diff
@@ -0,0 +1,76 @@
+from api.newsletters.newsletter_service import(address, get_subscription_list,add_to_subscription_list,check_subscription_list,remove_from_subscription_list)
+from api.newsletters.smtp import (send_newsletters,format_message)
+import json
+
+import logging
+from flask import (
+    Blueprint,
+    request
+)
+from api.security.guards import (
+    authorization_guard,
+    permissions_guard,
+    admin_messages_permissions
+)
+
+
+bp_name = 'api-newsletter'
+bp_url_prefix = '/api/newsletter'
+bp = Blueprint(bp_name, __name__, url_prefix=bp_url_prefix)
+
+logger = logging.getLogger("myapp")
+
+
+@bp.route("/")
+@authorization_guard
+@permissions_guard([admin_messages_permissions.read])
+def newsletter():
+    return get_subscription_list()
+
+
+@bp.route("/<user_id>")
+# @authorization_guard
+# @permissions_guard([admin_messages_permissions.read])
+def check_sub(user_id):
+    return check_subscription_list(user_id=user_id)
+
+@bp.route("/send_newsletter", methods=["POST"])
```

Comment:
@leonkoech  You don't need to repeat the noun here since you already have /api/newsletter, this endpoint should just be /api/newsletter/send (remove `_newsletter`)

Reply:
Question: I had the guard on the main newsletter route ie. `newsletter/`. Since /send_newsletter is newsletter/send_newsletter in full, does it still need the auth guard?

and yes, I agree! it is double descriptive. Will work on it @gregv